### PR TITLE
Add generic/generic_thermostat/homekit/min_max/history_stats/trend/ping/filesize to the list of reloadables

### PR DESCRIPTION
**Pull request by bdraco**
Backend PRs (not blocking since they won't show up until these are merged):
https://github.com/home-assistant/core/pull/39289
https://github.com/home-assistant/core/pull/39291
https://github.com/home-assistant/core/pull/39326
https://github.com/home-assistant/core/pull/39327
https://github.com/home-assistant/core/pull/39337
https://github.com/home-assistant/core/pull/39341
https://github.com/home-assistant/core/pull/39344
https://github.com/home-assistant/core/pull/39347
## Proposed change

Add generic/generic_thermostat/homekit/min_max/history_stats/trend/ping/filesize to the list of reloadables

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -843,7 +843,15 @@
               "rest": "Reload rest entities",
               "command_line": "Reload command line entities",
               "filter": "Reload filter entities",
-              "statistics": "Reload statistics entities"
+              "statistics": "Reload statistics entities",
+              "generic": "Reload generic IP camera entities",
+              "generic_thermostat": "Reload generic thermostat entities",
+              "homekit": "Reload homekit",
+              "min_max": "Reload min_max entities",
```
Comment (zsarnett) [inline src/translations/en.json:850]:
```suggestion
              "min_max": "Reload Min/Max entities",
```

File: src/translations/en.json [hunk 2]
```diff
@@ -843,7 +843,15 @@
...
+              "history_stats": "Reload history_stats entities",
```
Comment (zsarnett) [inline src/translations/en.json:851]:
```suggestion
              "history_stats": "Reload history stats entities",
```

File: src/translations/en.json [hunk 3]
```diff
@@ -843,7 +843,15 @@
...
+              "trend": "Reload trend entities",
+              "ping": "Reload ping binary sensor entities",
+              "filesize": "Reload filesize entities"
```
Comment (zsarnett) [inline src/translations/en.json:854]:
```suggestion
              "filesize": "Reload file size entities"
```

File: src/panels/config/server_control/ha-config-server-control.ts [hunk 4]
```diff
@@ -41,6 +41,14 @@ const reloadableDomains = [
   "command_line",
   "filter",
   "statistics",
+  "generic",
```
Comment (zsarnett) [inline src/panels/config/server_control/ha-config-server-control.ts:44]:
```suggestion
  "generic_camera",
```

File: src/translations/en.json [hunk 5]
```diff
@@ -843,7 +843,15 @@
               "rest": "Reload rest entities",
               "command_line": "Reload command line entities",
               "filter": "Reload filter entities",
-              "statistics": "Reload statistics entities"
+              "statistics": "Reload statistics entities",
+              "generic": "Reload generic IP camera entities",
```
Comment (zsarnett) [inline src/translations/en.json:847]:
```suggestion
              "generic_camera": "Reload generic IP camera entities",
```

File: src/translations/en.json [hunk 6]
```diff
@@ -843,7 +843,15 @@
...
+              "generic_thermostat": "Reload generic thermostat entities",
+              "homekit": "Reload homekit",
```
Comment (zsarnett) [inline src/translations/en.json:849]:
```suggestion
              "homekit": "Reload HomeKit",
```

File: src/panels/config/server_control/ha-config-server-control.ts (hunk 4 above)
Comment (bdraco) [reply src/panels/config/server_control/ha-config-server-control.ts:44]: Sadly, it actually has the domain `generic`

Comment (bdraco) [reply src/panels/config/server_control/ha-config-server-control.ts:44]: https://github.com/home-assistant/core/blob/dev/homeassistant/components/generic/camera.py

File: src/translations/en.json (hunk 5 above)
Comment (bdraco) [reply src/translations/en.json:847]: Please see above

**Review by bramkragten — APPROVED** [review]

**Pull request merged**